### PR TITLE
:book: "Guiding user stories" in contributor guidelines

### PR DESCRIPTION
**Pull request by jbalonso**
Document some guiding principles for the scope and ideals of the Kairos project for the purposes of shaping and organizing contributions.

**What this PR does / why we need it**:

The project backlog is now sprawling, such that it is difficult to readily tell what issues should be prioritized. This is an attempt to define values for the project that can help give structure to the planning of efforts and to help contributors have an intuition for what fits into the scope of the project.

The very concept of the list is a proposal, as well as the particular list. I recommend, though, not seeing the list as something... immutable.

### CODE REVIEW

File: CONTRIBUTING.md
Comment (mauromorales) [inline CONTRIBUTING.md:21]: Does k8s mean any version of Kubernetes, including k3s? Would it be better to say k3s, since that's what is provided? Or if we want to keep it open, maybe use the word Kubernetes? Please excuse the n00b question, I'm very new to the Kubernetes community, so I'm not sure what people understand when they read k8s.

**Review by mudler — APPROVED** [review]
Thanks @jbalonso ! it looks good here. However, I'd give a bit of more time before merging so the team can have a look at it cc: @kairos-io/maintainers

**Review by mauromorales — APPROVED** [review]

**Pull request merged**
